pkgs/tools/security/bitwarden/cli.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, buildNpmPackage
+, nodejs_18
+, fetchFromGitHub
+, python3
+}:
+
+let
+  buildNpmPackage' = buildNpmPackage.override { nodejs = nodejs_18; };
+in buildNpmPackage' rec {
+  pname = "bitwarden-cli";
+  version = "2023.5.0";
+
+  src = fetchFromGitHub {
+    owner = "bitwarden";
+    repo = "clients";
+    rev = "cli-v${version}";
+    hash = "sha256-ELKpGSY4ZbgSk4vJnTiB+IOa8RQU8Ahy3A1mYsKtthU=";
+  };
+
+  npmDepsHash = "sha256-G8DEYPjEP3L4s0pr5n2ZTj8kkT0E7Po1BKhZ2hUdJuY=";
+
+  nativeBuildInputs = [
+    python3
+  ];
```

```suggestion
  nativeBuildInputs = [
    python3
  ] ++ lib.optionals stdenv.isDarwin [
    darwin.cctools
  ];
```

Add this for `libtool` which node-gyp uses on darwin